Source/WebKit/NetworkProcess/webtransport/cocoa/NetworkTransportSessionCocoa.mm

```diff
@@ -32,11 +32,6 @@
 #import <pal/spi/cf/CFNetworkSPI.h>
 #import <wtf/RetainPtr.h>
 
-// FIXME: This is only needed for the internal Ventura build. Remove when appropriate.
```

Comment:
This looks wrong. Isn't the one in CFNetworkSPI.h only used for public builds, not internal ones? 

Of course it would be ideal to centralize SPI in the dedicated header, and to add a version check for macOS Ventura.